recipe/meta.yaml

```diff
@@ -36,7 +36,8 @@ test:
 about:
   home: http://jupyter.org
   license: BSD 3-clause
-  summary: the base implementation of the Jupyter Notebook format
+  summary: Reference implementation of the Jupyter Notebook format
```

Comment:
Seems it has been simplified to, "The Jupyter Notebook format".

ref: https://github.com/jupyter/nbformat/blob/4.3.0/setup.py#L63